source/ed/connectors/geopal_parser.cpp

```diff
@@ -382,6 +383,11 @@ void GeopalParser::fill_ways_edges(){
                     }
                     wy->type ="";
                     wy->uri = wayd_uri;
+                    if (row[visible] == "0") {
```

Comment:
I put the priority on visible field = true, if it is broken

Reply:
To be backward compatible, you might want to make sure the field `visible` exists. 
In the case where the field `visible` is missing, you'll end up with `row[-1]` => :hankey: 

```suggestion
                    if (has_col(visible, row) && row[visible] == "0") {
```

Reply:
Agree with @nonifier , can be nice to (hand-)test that it's backward compatible.

Reply:
Nice catch :heart: 
you are my heroes